Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,16 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const taskProps: { [key: string]: any; } = {MiniMatchLines: sourceFilter.length};
```

Comment:
nitpick: space after the brackets

{ MiniMatchLines: sourceFilter.length };

Reply:
Done